packages/http/src/validator/validators/__tests__/headers.spec.ts

```diff
@@ -3,6 +3,7 @@ import { query as registry } from '../../deserializers';
 import { HttpHeadersValidator } from '../headers';
 import * as validateAgainstSchemaModule from '../utils';
 import { assertRight, assertLeft } from '@stoplight/prism-core/src/__tests__/utils';
+import { some } from 'fp-ts/lib/Option';
```

Comment:
```suggestion
import * as Option from 'fp-ts/lib/Option';
```

Although `some` is pretty typical for FP I would stick to our convention.